src/library/Uno.Material/Styles/Controls/Button.xaml

```diff
@@ -842,23 +842,10 @@
 
 						<win:Button Content="See Style"
 									FontSize="10"
-									Style="{StaticResource TextMaterialButtonStyle}"
-									Margin="12,0">
+									Style="{StaticResource TextMaterialButtonStyle}">
 							<Button.Flyout>
 								<Flyout>
-									<ContentPresenter x:Name="ContentPresenter"
```

Comment:
I see why you replaced this but im not 100% convinced, maybe someone wants to add an icon or something in the button, it probably won't look super good but with a textblock it will crash.

Reply:
nevermind! Move this style to sample :D